fed/proxy/grpc/grpc_proxy.py

```diff
@@ -142,7 +142,7 @@ def get_grpc_config_by_party(self, dest_party):
         grpc_metadata = self._grpc_metadata
         grpc_options = self._grpc_options
 
-        dest_party_msg_config = self._cluster[dest_party].get(
+        dest_party_msg_config = self._addresses[dest_party].get(
```

Comment:
So we still support this party config in `addresses`?

Reply:
For now, yes.
But party-specific config will be deprecated soon or later.

Reply:
I deprecated it in this PR.